src/theme/numberInput.ts

```diff
@@ -0,0 +1,28 @@
+const numberInput = {
+  components: {
+    NumberInput: {
+      baseStyle: {
+        field: {
+          borderRadius: "5px",
+          marginLeft: "2px",
+          marginRight: "2px",
```

Comment:
replace with `marginX: 1`

Reply:
![Uploading Screenshot 2024-11-04 at 11.23.12.png…]()
doesn't work.

Reply:
It works okay if i write this in call.

Reply:
okay then